scripts/getNavForTranslation.js

```diff
@@ -0,0 +1,86 @@
+const fs = require('fs');
+const path = require('path');
+const yaml = require('js-yaml');
+
+const isFile = (filepath) => fs.statSync(filepath).isFile();
+const pathJoin = (base) => (filepath) => path.join(base, filepath);
+const skippedTitles = ['Style guide', 'root'];
+const locale = process.argv.slice(2)[0];
+
+const printExample = () => console.warn('Example: yarn generate-nav-titles jp');
+
+if (!locale) {
+  console.warn('<!> No locale specified, please append to the command');
+  printExample();
+  process.exit(0);
+}
+
+const getNavFiles = () => {
+  const rawFiles = fs
+    .readdirSync('src/nav')
+    .map(pathJoin('src/nav'))
+    .filter(isFile);
+
+  const parsedFiles = rawFiles.map((file) => {
+    const fileContents = fs.readFileSync(file, 'utf8');
+    const data = yaml.safeLoad(fileContents);
+    return data;
+  });
+  return parsedFiles;
+};
+
+const readSubNavs = (subNav, flatNav) => {
+  return subNav.flatMap((file) => {
+    const appendedNav = [
+      ...flatNav,
+      { englishTitle: file.title, title: file.title, locale },
+    ];
+    if (file.pages) {
+      return readSubNavs(file.pages, appendedNav);
+    } else {
+      return appendedNav;
+    }
+  });
+};
+
+const getFlatNav = () => {
```

Comment:
Should this take a param if we're passing it in on line 61?

Reply:
whoops nope, that was from before i made locale a global var